conan/cli/commands/audit.py

```diff
@@ -20,10 +20,30 @@ def _add_provider_arg(subparser):
     subparser.add_argument("-p", "--provider", help="Provider to use for scanning")
 
 
+def _parse_error_threshold(result: dict, error_level: float) -> None:
+    """Mark the result as error if any of the vulnerabilities has a severity greater than or equal to
+       The error_level reflects the severity level configured by the user. As it uses float and CVSS
+       score is limited to 10.0, users can use any higher number to skip it.
+
+       :param result: Conan audit scan result. It's expected to find cvss there
+       :param error_level: Threshold to raise an error in case of matching the severity level
+       :return: None
+    """
+    if "conan_error" not in result:
+        for ref in result["data"]:
+            for edge in result["data"][ref]["vulnerabilities"]["edges"]:
+                preferred_base_score = float(edge["node"]["cvss"].get("preferredBaseScore", 0.0))
+                if preferred_base_score >= error_level:
+                    result.update(
+                        {"conan_error":
+                             "The conan audit command exceeded the threshold severity level."})
```

Comment:
I think you should add here the `ref` the `score` for that ref and the limit `score`, because this message is going to be the part of an exception that is going to be raised after all output is printed, so if you have a large dependency tree is not clear which dependency is over the limit.